draft-irtf-cfrg-vdaf.md

```diff
@@ -3020,8 +3029,24 @@ removing it after unsharding.
 
 # IANA Considerations
 
-This document makes no request of IANA.
+A codepoint for each (V)DAF in this document is defined in the table below. Note
+that `0x00000000` through `0x000000FF` are reserved for testing purposes.
 
+| Value      | Scheme               | Type | Reference |
+|:-----------|:---------------------|:-----|:----------|
+| `0x000000XX` | reserved for testing | n/a  | n/a       |
```

Comment:
The more common way to specify test code points is as a range, so `0x00000000 - 0x000000ff` here (as in the prose above).

Also, usually a reservation like this is made not just for testing, but for "vendor specific usage" -- whatever you want to do for your own products that you don't expect to generally interop.

Reply:
Good catch. I'm also going to bump the range to 0x00000000 - 0x0000FFFF so that there's more room for "vendor specific usage".